### PR TITLE
22.10.2

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update to 22.10.2

https://github.com/gevent/gevent/blob/22.10.2/CHANGES.rst

https://github.com/gevent/gevent/blob/22.10.2/pyproject.toml

https://github.com/gevent/gevent/blob/22.10.2/setup.py

https://anaconda.atlassian.net/browse/PKG-880

### CODE REVIEW

**Review by ELundby45 — APPROVED** [review]
LGTM

**Review by varlackc — COMMENTED** [review]
We might want to be careful with the pinings in the `host` section

https://github.com/AnacondaRecipes/gevent-feedstock/blob/993cd59ecb07b6b9230dadc3412bfb382f9701dd/recipe/meta.yaml#L42

https://github.com/AnacondaRecipes/gevent-feedstock/blob/993cd59ecb07b6b9230dadc3412bfb382f9701dd/recipe/meta.yaml#L46

https://github.com/AnacondaRecipes/gevent-feedstock/blob/993cd59ecb07b6b9230dadc3412bfb382f9701dd/recipe/meta.yaml#L47

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ requirements:
     - {{ compiler('c') }}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:29]: As we do not build cross-platform these lines can be removed from `build`:
```
    - python                                 # [build_platform != target_platform]
    - cffi                                   # [build_platform != target_platform]
```

Comment (cbouss) [reply recipe/meta.yaml:29]: addressed

**Comment by cbouss** [comment]
> We might want to be careful with the pinings in the `host` section
> 
> https://github.com/AnacondaRecipes/gevent-feedstock/blob/993cd59ecb07b6b9230dadc3412bfb382f9701dd/recipe/meta.yaml#L42
> 
> https://github.com/AnacondaRecipes/gevent-feedstock/blob/993cd59ecb07b6b9230dadc3412bfb382f9701dd/recipe/meta.yaml#L46
> 
> https://github.com/AnacondaRecipes/gevent-feedstock/blob/993cd59ecb07b6b9230dadc3412bfb382f9701dd/recipe/meta.yaml#L47

Do you have suggestions?

**Comment by varlackc** [comment]
> > We might want to be careful with the pinings in the `host` section
> > https://github.com/AnacondaRecipes/gevent-feedstock/blob/993cd59ecb07b6b9230dadc3412bfb382f9701dd/recipe/meta.yaml#L42
> > 
> > https://github.com/AnacondaRecipes/gevent-feedstock/blob/993cd59ecb07b6b9230dadc3412bfb382f9701dd/recipe/meta.yaml#L46
> > 
> > https://github.com/AnacondaRecipes/gevent-feedstock/blob/993cd59ecb07b6b9230dadc3412bfb382f9701dd/recipe/meta.yaml#L47
> 
> Do you have suggestions?

In principle we want to avoid the comparison operators in the host section, however this might not be possible in all cases. 

In this particular case we might not have many good options, for now we might have to specify the earliest version that is supported in the host for `cffi`, and do some testing to see if the package still build correctly. 

This is just a suggestion, but some exceptions might be necessary in some situations, this might be one of those exceptions.

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**